### PR TITLE
Adjusting some places for onnx bc changes

### PR DESCRIPTION
**Pull request by bddppq**
https://github.com/onnx/onnx/pull/66

### CODE REVIEW

**Comment by bddppq** [comment]
Disabled running onnx backend test until the model files on aws are regenerated

File: tests/onnx_backend_test.py [hunk 1]
```diff
@@ -10,7 +10,7 @@
 import onnx_caffe2.backend as c2
 
 # unittest by default only looks for tests in current scope!
-_ = onnx.backend.test.BackendTest(c2).tests
+# _ = onnx.backend.test.BackendTest(c2).tests
```
Comment (ezyang) [inline tests/onnx_backend_test.py]: I guess, broken for now?

File: onnx_caffe2/frontend.py [hunk 2]
```diff
@@ -271,14 +279,13 @@ def caffe2_init_net_to_initializers(init_net):
...
+    return make_model(caffe2_net_to_onnx_graph(net_def))
+
+
 def caffe2_net_to_onnx_graph(net_def):
-    if not net_def:
-        return None
     graph_def = onnx_pb2.GraphProto()
 
-    # to pass the check HasField('version')
-    graph_def.ir_version = onnx_pb2.IR_VERSION
```
Comment (ezyang) [inline onnx_caffe2/frontend.py]: I don't see any setting of ir_version in model in this diff. We should set it!

File: tests/onnx_backend_test.py (hunk 1 above)
Comment (ezyang) [reply tests/onnx_backend_test.py]: OK I see, this is the model tests. I wasn't sure from looking at the code.

File: onnx_caffe2/frontend.py (hunk 2 above)
Comment (bddppq) [reply onnx_caffe2/frontend.py]: It's automatically set in `make_model` https://github.com/onnx/onnx/blob/master/onnx/helper.py#L43

File: onnx_caffe2/frontend.py [hunk 3]
```diff
@@ -261,7 +269,7 @@ def caffe2_init_net_to_initializers(init_net):
         if raw:
             assert np_type
             vals = np.asarray(vals, dtype=np_type).tobytes()
-        initializers.append(helper.make_tensor(
+        initializers.append(make_tensor(
```
[the remark this answers is not in the document]
Comment (bddppq) [reply onnx_caffe2/frontend.py]: Sure will fully qualify it

File: onnx_caffe2/backend.py [hunk 4]
```diff
@@ -521,8 +513,8 @@ def fresh(self):
         return net_def, env
 
 
-run_node = Caffe2Backend.run_node
-
 prepare = Caffe2Backend.prepare
 
-run_graph = Caffe2Backend.run_graph
```
[the remark this answers is not in the document]
Comment (bddppq) [reply onnx_caffe2/backend.py]: Let's not :-)
`run_graph`/`run_model` is not expected to be used very often anyway. The "official" guide is to use `prepare` + `run`.

**Pull request merged**